endpoint/endpoint_core.c

```diff
@@ -54,14 +46,18 @@ void endpoint_destroy() {
   logger_helper_destroy();
 }
 
-status_t send_transaction_information(int value, const char* message, const char* message_fmt, const char* tag,
+status_t send_transaction_information(const char* host, const char* port, const char* ssl_seed, int value,
+                                      const char* message, const char* message_fmt, const char* tag,
                                       const char* address, const char* next_address, const uint8_t* private_key,
                                       const char* device_id, uint8_t* iv) {
   char tryte_msg[MAX_MSG_LEN] = {0};
   char msg[MAX_MSG_LEN] = {0};
   char req_body[MAX_MSG_LEN] = {0};
   uint8_t ciphertext[MAX_MSG_LEN] = {0};
   uint8_t raw_msg[MAX_MSG_LEN] = {0};
+  const char* ta_host = host ? host : "localhost";
```

Comment:
Let these values be build-time configurable. That is, place some macros which indicate the default values, such as DEF_TA_PORT.

Reply:
The macro has been reverted.